src/main/java/com/spotinst/sdkjava/model/converters/OceanEcsLaunchSpecConverter.java

```diff
@@ -0,0 +1,486 @@
+        ClusterDynamicVolumeSize retVal = null;
+
+        if (dynamicVolumeSize != null) {
+            ClusterDynamicVolumeSize.Builder volumeSizeBuilder = ClusterDynamicVolumeSize.Builder.get();
+
+            if (dynamicVolumeSize.isBaseSizeSet()) {
+                volumeSizeBuilder.setBaseSize(dynamicVolumeSize.getBaseSize());
+            }
+            if (dynamicVolumeSize.isResourceSet()) {
+                volumeSizeBuilder.setResource(dynamicVolumeSize.getResource());
+            }
+            if (dynamicVolumeSize.isSizePerResourceUnitSet()) {
+                volumeSizeBuilder.setSizePerResourceUnit(dynamicVolumeSize.getSizePerResourceUnit());
+            }
+            retVal = volumeSizeBuilder.build();
+        }
+        return retVal;
+    }
+
+    private static ClusterIamInstanceProfileSpec toBl(ApiClusterIamInstanceProfileSpec instanceProfile) {
+        ClusterIamInstanceProfileSpec retVal = null;
+
+        if (instanceProfile != null) {
+            ClusterIamInstanceProfileSpec.Builder IamInstanceProfileSpecBuilder =
+                    ClusterIamInstanceProfileSpec.Builder.get();
+
+            if (instanceProfile.isArnSet()) {
+                IamInstanceProfileSpecBuilder.setArn(instanceProfile.getArn());
+            }
+            if (instanceProfile.isNameSet()) {
+                IamInstanceProfileSpecBuilder.setName(instanceProfile.getName());
+            }
+            retVal = IamInstanceProfileSpecBuilder.build();
+        }
+        return retVal;
+    }
+
+    private static ClusterInstanceMetadataOptions toBl(ApiClusterInstanceMetadataOptions instanceMetadataOptions) {
+        ClusterInstanceMetadataOptions retVal = null;
+
+        if (instanceMetadataOptions != null) {
+            ClusterInstanceMetadataOptions.Builder instanceMetadataOptionsBuilder =
+                    ClusterInstanceMetadataOptions.Builder.get();
+
+            if (instanceMetadataOptions.isHttpPutResponseHopLimitSet()) {
+                instanceMetadataOptionsBuilder
+                        .setHttpPutResponseHopLimit(instanceMetadataOptions.getHttpPutResponseHopLimit());
+            }
+            if (instanceMetadataOptions.isHttpTokensSet()) {
+                instanceMetadataOptionsBuilder.setHttpTokens(instanceMetadataOptions.getHttpTokens());
+            }
+            retVal = instanceMetadataOptionsBuilder.build();
+        }
+        return retVal;
+    }
+
+    private static ClusterTagsSepcification toBl(ApiClusterTagsSepcification tag) {
+        ClusterTagsSepcification retVal = null;
+
+        if (tag != null) {
+            ClusterTagsSepcification.Builder tagBuilder = ClusterTagsSepcification.Builder.get();
+
+            if (tag.isTagKeySet() && tag.isTagValueSet()) {
+                tagBuilder.setTagKey(tag.getTagKey());
+                tagBuilder.setTagValue(tag.getTagValue());
+            }
+            retVal = tagBuilder.build();
+        }
+        return retVal;
+    }
+}
```

Comment:
Move this file to ocean/ecs folder inside converters folder, so that proper hierarchy is maintained for new code.